schema/mzqc_schema.json

```diff
@@ -147,6 +159,10 @@
                         ]
                     }
                 },
+                "labelName": {
+                    "description": "OPTIONAL label name. For setQuality, this a group name, lending itself for example as a axis labels for a plot. OPTIONAL.",
```

Comment:
we need to discuss how this relates to inputFile:name

Reply:
I'd suggest to describe it in the spec doc with something like "We suggest to use labelName only if either a label different from the inputFile.name needs to be specified or grouping labels are wanted. In case references for a setQuality are needed, labelName needs to be used."

Reply:
Maybe "label" rather than "labelName". The latter is kind of two times the same thing?

Reply:
true :smile: dubbelop  
`label` then?